package-lock.json

```diff
@@ -18734,9 +18599,9 @@
       },
       "dependencies": {
         "@babel/runtime": {
-          "version": "7.6.2",
-          "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.6.2.tgz",
-          "integrity": "sha512-EXxN64agfUqqIGeEjI5dL5z0Sw0ZwWo1mLTi4mQowCZ42O59b7DRpZAnTC6OqdF28wMBMFKNb/4uFGrVaigSpg==",
+          "version": "7.6.3",
```

Comment:
This seems like it upgrades a lot of unrelated stuff. Can we double check that?

Reply:
I corrected this. I should have checked the lock diff before submitting the PR - thanks!